benchmark/benchmark_json/include/SerializationBenchmarkUtils.h

```diff
@@ -0,0 +1,8 @@
+#ifndef SERIALIZATIONBENCHMARKUTILS_H
+#define SERIALIZATIONBENCHMARKUTILS_H
+
+#include <iostream>
+
+float get_random_float_using_whole_range();
```

Comment:
Name and make it a member of a class with static methods.

Reply:
Done.